### PR TITLE
Fix: properly handle monochrome devices

### PR DESCRIPTION
**Pull request by SimonBrandner**
Improves https://github.com/AaronErhardt/tuxedo-rs/issues/32

<hr>

This PR fixes a bug where if one uses `tailord` with a device that doesn't support RGB, `tailord` would still default to an RGB profile which changes over time, this would then reset the keyboard backlight each time the user tried to change it manually.

The issue is fixed by making the default `ColorProfile` dependent on the `LedControllerMode` (i.e. its support of RGB)

<hr>

Notes:
- This is my first Rust PR: hopefully, it doesn't suck too much :sweat_smile: 
- I am not 100% confident this is the right way to fix things - I am happy to improve this in any way

### CODE REVIEW

**Comment by SimonBrandner** [comment]
(for anyone interested in getting this running before it is merged: https://github.com/SimonBrandner/dotfiles/commit/f51ec63b3cabc09dd12161e8a925711cdddf4b70)

**Comment by F3l1XQu1ll** [comment]
Tested on nixos-unstable according to https://github.com/SimonBrandner/dotfiles/commit/f51ec63b3cabc09dd12161e8a925711cdddf4b70 
Controlling the keyboard backlight on my InfinityBook Pro 16 Gen 8 now works like a charm with all three brightness levels using the keyboard shortcut as well as with the controls in GNOME (although the settings in GNOME don't reflect the current state of the backlight, but this seems to be a problem with GNOME)

File: tailord/src/main.rs [hunk 1]
```diff
@@ -40,11 +42,24 @@ fn main() {
 
 #[tracing::instrument]
 async fn start_runtime() {
+    println!("start_runtime()");
```
Comment (AaronErhardt) [inline tailord/src/main.rs:45]: Please remove the `println!`

**Review by AaronErhardt — CHANGES_REQUESTED** [review]
Looks good except for the left over `println!`.

Also maybe the `match` statements over the `find_map` iterator could be simplified using `unwrap_or_else`.

**Comment by SimonBrandner** [comment]
> Looks good except for the left over `println!`.

Oops  :sweat_smile: 

> Also maybe the `match` statements over the `find_map` iterator could be simplified using `unwrap_or_else`.

New Rust feature for me - thanks for the pointer - I was hoping there'd be something like this!

**Review by AaronErhardt — APPROVED** [review]
Looks good. Thanks a lot!

**Pull request merged**